.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,50 @@
+---
+name: 'Libp2p Release'
+about: 'Start a new libp2p release.'
+---
+
+## 🗺 What's left for release
+
+<List of items with remaining PRs and/or Issues to be considered for this release>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by scripts/mkreleaselog >
+
+## ✅ Release Checklist
+
+- [ ] **Stage 0 - Automated Testing**
+    - [ ] Fork a new `release-vX.Y.Z` branch from `master`, freezing master.
+    - [ ] Make sure local tests are passing.
+- [ ] **Stage 1 - Upstream Testing**
+  - Create testing branches in upstream repos with the new go-libp2p release and run CI/tests.
+    - [ ] [filecoin-project/lotus](https://github.com/filecoin-project/lotus)
+    - [ ] [ipfs/go-bitswap](https://github.com/ipfs/go-bitswap)
+    - [ ] [ipfs/go-ipfs](https://github.com/ipfs/go-ipfs)
+    - [ ] [libp2p/go-libp2p-kad-dht](https://github.com/libp2p/go-libp2p-kad-dht)
+    - [ ] [libp2p/go-libp2p-pubsub](https://github.com/libp2p/go-libp2p-pubsub)
+  - _(someday)_ Run upstream testground tests. Unfortunately, this is too time consuming at the moment.
+    - _(someday)_ Run bitswap testground tests.
+    - _(someday)_ Run DHT testground tests.
+- [ ] **Stage 2 - Infrastructure Testing**
```

Comment:
Whose responsibility is this? Do we need to get the infrastructure team involved at that point?

Reply:
We're responsible for creating the appropriate branches, infra is responsible for deploying.